alacritty_terminal/src/term/mod.rs

```diff
@@ -2755,6 +2779,43 @@ mod tests {
         assert_eq!(version_number("1.2.3-dev"), 1_02_03);
         assert_eq!(version_number("999.99.99"), 9_99_99_99);
     }
+
+    #[test]
+    fn report_size_in_pixels_and_chars() {
```

Comment:
I'm not entirely sure about the stuff this test covers. If the goal is just to cover the dispatched method we shouldn't run a parser. If the goal is to test that the handler dispatches the correct method then we should move it to `ansi.rs` and don't test what the handler itself does.

So it seems like this would better be separated, to test only one specific thing rather than two. Most of the `ansi.rs` isn't covered but I think that's not too big of a deal, since the stuff is mostly plug and play and there's not a lot of functionality to cover with tests. There's no need to cover vte either way.

Reply:
I wanted a test to cover both the parser and the handler of the sequence. I removed it 34d57a289e9096605230bd585cce884ee2b7d847.